package.json

```
@@ -101,10 +102,16 @@
   "devDependencies": {
     "@babel/core": "^7.4.3",
     "@babel/preset-env": "^7.4.3",
+    "@wdio/browser-runner": "^8.10.4",
+    "@wdio/cli": "^8.10.4",
+    "@wdio/concise-reporter": "^8.10.4",
+    "@wdio/globals": "^8.11.3",
+    "@wdio/mocha-framework": "^8.10.4",
     "babel-loader": "^8.0.5",
     "c8": "^7.12.0",
     "chai": "^4.0.1",
     "cheerio": "^0.22.0",
+    "chromedriver": "^114.0.0",
```

```suggestion
    "chromedriver": "^106.0.0",
```

it should be fixed if back to the version: https://github.com/aladdin-add/eslint/actions/runs/5571225831/jobs/10176049189?pr=21